manifests/t/TerraformLinters/tflint/0.44.1/TerraformLinters.tflint.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: TerraformLinters.tflint
+PackageVersion: 0.44.1
+InstallerLocale: en-US
+InstallerType: zip
+Commands:
+- tflint
+FileExtensions:
+- hcl
+- tf
+ReleaseDate: 2022-12-29
```

```suggestion
ReleaseDate: 2022-12-30
```
As per the [ReleaseNotes](https://github.com/terraform-linters/tflint/releases/tag/v0.44.1)